srcalc/src/main/java/gov/va/med/srcalc/web/controller/CalculationController.java

```diff
@@ -111,9 +111,10 @@ public ModelAndView presentVariableEntry(
         // Present the view.
         final ModelAndView mav = new ModelAndView(Tile.ENTER_VARIABLES);
         mav.addObject("calculation", workflow.getCalculation());
+        mav.addObject("selectedProcedure", initialValues.getDynamicValues().get("Procedure"));
```

Comment:
This is ugly. The data is already in the model via the "variableEntry" attribute.